content/en/docs/reference/glossary/kube-scheduler.md

```diff
@@ -4,15 +4,20 @@ id: kube-scheduler
 date: 2018-04-12
 full_link: /docs/reference/generated/kube-scheduler/
 short_description: >
-  Control Plane component that watches for newly created pods with no assigned node, and selects a node for them to run on.
+  Control plane component that watches for newly created pods with no assigned node, and selects a node for them to run on.
 
 aka: 
 tags:
 - architecture
 ---
- Control Plane component that watches for newly created pods with no assigned node, and selects a node for them to run on.
+Control plane component that watches for newly created
+{{< glossary_tooltip term_id="node" >}} with no assigned
```

Comment:
@sftim , going through a final read. Is line 14 correct? Should it be, `newly created Pods` with no assigned node, ...

Reply:
D'oh!